### PR TITLE
Simple help command UI update

### PR DESCRIPTION
**Pull request by Ganoosh**

### CODE REVIEW

File: src/cogs/help.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Mainly a help command that replaces the default command that discord.py provides.
 This module also contains related commands and functions such as about, contrib, docs.
 """
+from turtle import color
```
Comment (awsomearvinder) [inline src/cogs/help.py]: Why's this here?

File: src/cogs/help.py [hunk 2]
```diff
@@ -97,10 +120,7 @@ def command_not_found(self, string: str) -> str:
             key=lambda command: -fuzz.ratio(command.name, attempted_command),
         )
         new_line = "\n"
```
Comment (awsomearvinder) [inline src/cogs/help.py:122]: this variable is no longer needed because of your changes

File: src/cogs/help.py [hunk 3]
```diff
@@ -97,10 +120,7 @@ def command_not_found(self, string: str) -> str:
             key=lambda command: -fuzz.ratio(command.name, attempted_command),
```
Comment (awsomearvinder) [inline src/cogs/help.py:120]: We could probably use min() now instead of sorted I think?

File: src/cogs/help.py [hunk 4]
```diff
@@ -40,33 +40,50 @@ def get_bot_help(
...
+                out.add_field(
+                    name=cog.qualified_name,
+                    value=f'`{count(cog.walk_commands())} {"command" if count(cog.walk_commands()) == 1 else "commands"}`',
+                    inline=True,
+                )
+            else:
+                freestanding_commands.extend(cmds)
+
+        if len(out.fields) % 3 == 2:
+            out.add_field(name="\u200b", value="\u200b", inline=True)
```
Comment (awsomearvinder) [inline src/cogs/help.py:66]: This seems kind of opaque to me, some comments here would be nice

File: src/cogs/help.py [hunk 5]
```diff
@@ -75,16 +92,22 @@ def get_cog_help(cog: commands.Cog) -> discord.Embed:
                 )
                 continue
             out.add_field(
-                name=command_usage(cmd_or_group),
-                value=(cmd_or_group.help if cmd_or_group.help else ""),
+                name=cmd_usage,
+                value=(cmd_or_group.help if cmd_or_group.help else "")
+                + f'\n```ini\n{cmd_usage if cmd_or_group.signature == None else cmd_usage + "[None]"}```',
```
Comment (awsomearvinder) [inline src/cogs/help.py:97]: `if cmd_or_group.signature is None`, more consistent, + pylint catches that.

File: src/cogs/help.py [hunk 6]
```diff
@@ -40,33 +40,50 @@ def get_bot_help(
         mapping: Mapping[Optional[commands.Cog], List[commands.Command]]
     ) -> discord.Embed:
         """Generates the embed for the bot help command."""
-        out = discord.Embed(title="Help")
+        out = discord.Embed(
+            title="Commands and modules",
+            description="View a specific module or commands info with:\n```ini\n!help [module_name|command_name]```",
```
Comment (awsomearvinder) [inline src/cogs/help.py:45]: Some strings like the one here are too long, but black in repo is broken right now, I'll probably fix this myself.

**Review by awsomearvinder — CHANGES_REQUESTED** [review]

**Comment by awsomearvinder** [comment]
Will merge after some testing tomorrow assuming issues above are fixed

**Comment by Ganoosh** [comment]
All requested changes have been made

**Pull request closed without merging**